ethcore/src/tests/client.rs

```diff
@@ -107,24 +107,6 @@ fn imports_good_block() {
 	assert!(!block.into_inner().is_empty());
 }
 
-#[test]
-fn fails_to_import_block_with_invalid_rlp() {
```

Comment:
this test is not needed, cause `import_block` takes already deserialized block as a param